DependenSee/ResultWriter.cs

```diff
@@ -54,11 +56,13 @@ internal void Write(DiscoveryResult result, OutputTypes type, string outputPath)
             }
         }
 
-        private void WriteAsHtmlToFile(DiscoveryResult result, string outputPath)
+        private void WriteAsHtmlToFile(DiscoveryResult result, string outputPath, string title)
         {
             var templatePath = Path.Combine(new FileInfo(Assembly.GetExecutingAssembly().Location).DirectoryName, HtmlTemplateName);
```

Comment:
Hey thanks for this. That was fast.

think we need to default to previous behavior if a title wasn't provided.

```suggestion
            var templatePath = Path.Combine(new FileInfo(Assembly.GetExecutingAssembly().Location).DirectoryName, HtmlTemplateName);
            title = string.IsNullOrWhiteSpace(title) ? "DependenSee" : title;
```

Reply:
Is it not already effectively doing that with the existing code? I thought that value might be more appropriate as a default for the parameter in `PowerArgsProgram.cs` rather than a constant / default in `ResultWriter.cs` because it wasn't directly related to writing results.

This way, if the title isn't specified, "DependenSee" is used, but it's also possible for the user to explicitly specify an empty string by specifying `-HtmlTitle ""` on the command line.

Leveraging PowerArgs like this makes it so we don't have to maintain that value in more than one place and also prevents the value from ever being `null` because of the default value and the fact that the user can't specify `null` on the command line (so one less thing to check for).

If it's just a matter of style and organization and this is the way you want it, I'm happy to change it (after all, it's your project). Just let me know and I'll do some rework.

Reply:
gotchya.. yea my bad. didn't notice the PowerArgs attribute. Will test and merge soon . 😊